src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3985,6 +4082,109 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if (( HybridModelZone( ZoneNum ).InternalThermalMassCalc || HybridModelZone( ZoneNum ).InfiltrationCalc ) && !WarmupFlag && !DoingSizing ) {
```

Comment:
Updated the statement as recommended. The higher level if statement improved the simulation speed couple of seconds.